src/App.tsx

```diff
@@ -192,7 +192,7 @@ function App() {
               <h2>Add Vacation!</h2>
               <div className="pair">
                 <label htmlFor="start">Vacation Start</label>
-                <input type="date" name="start" id="start" onChange={handleStartEndChange}/>
+                  <input type="date" min={new Date().toISOString().split('T')[0]} name="start" id="start" onChange={handleStartEndChange}/>
```

Comment:
Same suggestions/questions here. I would put this into a const and sort out if the splitting is necessary or not.

```suggestion
const today = new Date().toISOString


<input type="date" min={today.split('T')[0]} name="start" id="start" onChange={handleStartEndChange}/>
```